apps/dash-manufacture-spc-dashboard/assets/spc-custom-styles.css

```diff
@@ -0,0 +1,794 @@
+/* Table of contents
+––––––––––––––––––––––––––––––––––––––––––––––––––
+- Banner
+- Modal
+- Tabs
+- Main Dashboard Tab
+- Measurement Tab
+- Tables/Dropdown
+- Containers
+- Media Queries
+–––––––––––––––––––––––––––––––––––––––––––––––––– */
+
+* {
+  box-sizing: border-box;
+  -moz-box-sizing: border-box;
+}
+
+body {
+  background-color: #1e2130;
+  color: #f3f5f4;
+  font-family: "Open Sans", sans-serif;
+  width: 100%;
+  height: 100vh;
+  max-width: 100% !important;
+  overflow-x: hidden;
+  margin: 0;
+}
+
+#big-app-container {
+  max-width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0 6rem;
+}
+
+.banner {
+  height: fit-content;
+  background-color: #1e2130;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  border-bottom: darkgray solid 1px;
+  padding: 1rem 4rem;
```

Comment:
Along with other banner changes I mention, if you change the padding to `padding: 1rem 10rem;` and remove the following snippet in the css:
```
* {
  box-sizing: border-box;
  -moz-box-sizing: border-box;
}
```
I notice that the banner adjusts to look like the mock-up as follows:
<img width="1680" alt="Screen Shot 2019-07-23 at 16 07 13" src="https://user-images.githubusercontent.com/23238764/61743807-fa0c7a00-ad63-11e9-8d8f-8f9786b1ed99.png">

However, I don't know if you did this intentionally!

Reply:
this should be fixed. - banner extend to full width. thanks!